src/views/Step3Container.vue

```diff
@@ -0,0 +1,16 @@
+<!--Container for share structure or rules depending on the type of entity-->
```

Comment:
I think it would be better to have separate views for each entity type.

Reply:
So most of the top level pages are just containers for holding the child components. Max it does is pass the state as prop to the child component. I think having a top level container will hold and limit the logic with in that view

Reply:
But what if one entity type has 5 steps and another entity type has 6 steps?

Reply:
Ya sure. That is one of the advantage of retaining path in the step as well. If this has to be separate views you the step 3 'to' property in the step array can be modified as  to:getters.isTypeBcomp ? '/create-share-structure':'/create-rules' and there can be two routes pointing to two different views both with step as 3 in meta.

If one entity has different number of steps the step array returned would be different altogether and the same logic as above can be applied.

Reply:
OK